app/modules.go

```diff
@@ -312,6 +317,7 @@ func orderEndBlockers() []string {
 		poolstypes.ModuleName,
 		restakingtypes.ModuleName,
 		liquidvestingtypes.ModuleName,
+		gaspricetypes.ModuleName,
```

Comment:
Same here, we don't have end block logic for `gasprice` module, so we should omit it.